handlers/clickmobile/clickmobile.go

```diff
@@ -0,0 +1,160 @@
+package clickmobile
+
+import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/nyaruka/courier/utils/dates"
+)
+
+var (
+	sendURL      = "http://clickmobile.com/sendsms"
```

Comment:
Not yet ready, I am waiting for the proper URL here or make that configurable from the website claim page